extension/app/ts/utils/constants.ts

```diff
@@ -16,11 +16,11 @@ export const ERC20_APPROVAL_4BYTES = 0x095ea7b3
 export const ERC721_APPROVAL_FOR_ALL_4BYTES = 0xa22cb465
 
 // common event log signatures
-export const TRANSFER_LOG = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('Transfer(address,address,uint256)'))
-export const APPROVAL_LOG = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('Approval(address,address,uint256)'))
-export const ERC721_APPROVAL_FOR_ALL_LOG = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('ApprovalForAll(address,address,bool)'))
-export const DEPOSIT_LOG = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('Deposit(address,uint256)'))
-export const WITHDRAWAL_LOG = ethers.utils.keccak256(ethers.utils.toUtf8Bytes('Withdrawal(address,uint256)'))
+export const TRANSFER_LOG = ethers.keccak256(ethers.toUtf8Bytes('Transfer(address,address,uint256)'))
+export const APPROVAL_LOG = ethers.keccak256(ethers.toUtf8Bytes('Approval(address,address,uint256)'))
+export const ERC721_APPROVAL_FOR_ALL_LOG = ethers.keccak256(ethers.toUtf8Bytes('ApprovalForAll(address,address,bool)'))
+export const DEPOSIT_LOG = ethers.keccak256(ethers.toUtf8Bytes('Deposit(address,uint256)'))
+export const WITHDRAWAL_LOG = ethers.keccak256(ethers.toUtf8Bytes('Withdrawal(address,uint256)'))
```

Comment:
Does `ethers` not have a function to do this?  In particular, this requires that you correctly formulate the canonical version of the event signature but it would be better if we used a function that handled that for us so we could pass in a non-canonical version like `Transfer(address from, address to, uint256 amount)`.